Test_November12/ThreadSafePriorityQueue.cs

```diff
@@ -0,0 +1,53 @@
+namespace Test_November12;
+
+public class ThreadSafePriorityQueue<T, TP> where TP : IComparable
```

Comment:
```suggestion
public class ThreadSafePriorityQueue<Key, Priority> where Priority : IComparable
```
Имена типовых переменных - это имена. Поэтому они должны быть содержательными.

Reply:
-0.5

Reply:
-0.5